[BUG] Cannot generate build system for dynamic_data_nested_structs & dynamic_data_sequences

<!-- :warning: Please, try to follow the template -->

### System information

- **RTI Product**: Connext DDS
- **Version**: 6.1.0
- **Operating system**: Linux
- **Compiler**: GCC
- **Compiler version**: 7.5.0
- **Additional information**: N/A

### What is the current behavior?

There is an error when generating the build system:

```plaintext
CMake Error at CMakeLists.txt:13 (add_executable):
  Target "dynamic_data_nested_struct_cxx2" links to target
  "RTIConnextDDS::cpp2_api" but the target was not found.  Perhaps a
  find_package() call is missing for an IMPORTED target, or an ALIAS target
  is missing?
```

### Steps to reproduce the issue

Execute this sequence of orders in the example's repository:

```sh
mkdir build
cd build
cmake ..
```

### Expected behavior

It should generate the build system.

### Suggested solutions

Add find_package to the example.

--- a/examples/connext_dds/dynamic_data_nested_structs/c++/CMakeLists.txt
+++ b/examples/connext_dds/dynamic_data_nested_structs/c++/CMakeLists.txt
@@ -10,6 +10,16 @@ set(CMAKE_MODULE_PATH
     "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
 )
 
+# Find the RTI Connext DDS libraries
+if(NOT RTIConnextDDS_FOUND)
+    find_package(RTIConnextDDS
+        "6.1.0"
+        REQUIRED
+        COMPONENTS
+            core
+    )
+endif()
+
 add_executable(dynamic_data_nested_struct_cxx
     "${CMAKE_CURRENT_SOURCE_DIR}/dynamic_data_nested_struct_example.cxx"
 )

--- a/examples/connext_dds/dynamic_data_nested_structs/c++03/CMakeLists.txt
+++ b/examples/connext_dds/dynamic_data_nested_structs/c++03/CMakeLists.txt
@@ -10,6 +10,16 @@ set(CMAKE_MODULE_PATH
     "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
 )
 
+# Find the RTI Connext DDS libraries
+if(NOT RTIConnextDDS_FOUND)
+    find_package(RTIConnextDDS
+        "6.1.0"
+        REQUIRED
+        COMPONENTS
+            core
+    )
+endif()
+
 add_executable(dynamic_data_nested_struct_cxx2
     "${CMAKE_CURRENT_SOURCE_DIR}/dynamic_data_nested_struct_example.cxx"
 )

--- a/examples/connext_dds/dynamic_data_nested_structs/c/CMakeLists.txt
+++ b/examples/connext_dds/dynamic_data_nested_structs/c/CMakeLists.txt
@@ -10,6 +10,16 @@ set(CMAKE_MODULE_PATH
     "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
 )
 
+# Find the RTI Connext DDS libraries
+if(NOT RTIConnextDDS_FOUND)
+    find_package(RTIConnextDDS
+        "6.1.0"
+        REQUIRED
+        COMPONENTS
+            core
+    )
+endif()
+
 add_executable(dynamic_data_nested_struct_c
     "${CMAKE_CURRENT_SOURCE_DIR}/dynamic_data_nested_struct_example.c"
 )

--- a/examples/connext_dds/dynamic_data_sequences/c++/CMakeLists.txt
+++ b/examples/connext_dds/dynamic_data_sequences/c++/CMakeLists.txt
@@ -10,6 +10,16 @@ set(CMAKE_MODULE_PATH
     "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
 )
 
+# Find the RTI Connext DDS libraries
+if(NOT RTIConnextDDS_FOUND)
+    find_package(RTIConnextDDS
+        "6.1.0"
+        REQUIRED
+        COMPONENTS
+            core
+    )
+endif()
+
 add_executable(dynamic_data_sequences_cxx
     "${CMAKE_CURRENT_SOURCE_DIR}/dynamic_data_sequences.cxx"
 )

--- a/examples/connext_dds/dynamic_data_sequences/c++03/CMakeLists.txt
+++ b/examples/connext_dds/dynamic_data_sequences/c++03/CMakeLists.txt
@@ -10,6 +10,16 @@ set(CMAKE_MODULE_PATH
     "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
 )
 
+# Find the RTI Connext DDS libraries
+if(NOT RTIConnextDDS_FOUND)
+    find_package(RTIConnextDDS
+        "6.1.0"
+        REQUIRED
+        COMPONENTS
+            core
+    )
+endif()
+
 add_executable(dynamic_data_sequences_cxx2
     "${CMAKE_CURRENT_SOURCE_DIR}/dynamic_data_sequences.cxx"
 )

--- a/examples/connext_dds/dynamic_data_sequences/c/CMakeLists.txt
+++ b/examples/connext_dds/dynamic_data_sequences/c/CMakeLists.txt
@@ -10,6 +10,16 @@ set(CMAKE_MODULE_PATH
     "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
 )
 
+# Find the RTI Connext DDS libraries
+if(NOT RTIConnextDDS_FOUND)
+    find_package(RTIConnextDDS
+        "6.1.0"
+        REQUIRED
+        COMPONENTS
+            core
+    )
+endif()
+
 add_executable(dynamic_data_sequences_c
     "${CMAKE_CURRENT_SOURCE_DIR}/dynamic_data_sequences.c"
 )

